### DISCUSSION TITLE
Code differences

### DISCUSSION THREAD

**Discussion by kodyli** [laravel/framework#41674] [Q&A]
https://github.com/laravel/framework/blob/7e82e1a272a72d75b4cd7f35a2267e9af5d642c2/src/Illuminate/Routing/RouteRegistrar.php#L90

in my project it is:

    /**
     * Set the value for a given attribute.
     *
     * @param  string  $key
     * @param  mixed  $value
     * @return $this
     *
     * @throws \InvalidArgumentException
     */
    public function attribute($key, $value)
    {
        if (! in_array($key, $this->allowedAttributes)) {
            throw new InvalidArgumentException("Attribute [{$key}] does not exist.");
        }

        if ($key === 'middleware') {
            foreach ($value as $index => $middleware) {
                $value[$index] = (string) $middleware;
            }
        }

        $attributeKey = Arr::get($this->aliases, $key, $key);

        if ($key === 'withoutMiddleware') {
            $value = array_merge(
                (array) ($this->attributes[$attributeKey] ?? []), Arr::wrap($value)
            );
        }

        $this->attributes[$attributeKey] = $value;

        return $this;
    }

anything wrong?

**Comment by mikeydevelops** [comment]
Well, it seems that your project is using Laravel 9.x and the snippet linked is from version 6.20.44.

Nothing is wrong, you are just using probably the latest Laravel version.
